src/HealthcareApis/HealthcareApis/help/Get-AzHealthcareApisService.md

```diff
@@ -0,0 +1,245 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.HealthcareApisService.dll-Help.xml
+Module Name: Az.HealthcareApisService
+online version: https://docs.microsoft.com/en-us/powershell/module/az.healthcareapisservice/get-azhealthcareapisservice
+schema: 2.0.0
+---
+
+# Get-AzHealthcareApisService
+
+## SYNOPSIS
+Get the metadata of a service instance.
+
+## SYNTAX
+
+### ListParameterSet (Default)
+```
+Get-AzHealthcareApisService [-ResourceGroupName <String>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### ServiceNameParameterSet
+```
+Get-AzHealthcareApisService -ResourceGroupName <String> -Name <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceIdParameterSet
+```
+Get-AzHealthcareApisService -ResourceId <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Gets existing healthcareApis fhir service accounts created within the specified subscription or a resource group.
+
+## EXAMPLES
+
+### Example 1
```

Comment:
Example output looks a little weird in markdown. You can take this example:
https://github.com/Azure/azure-powershell/blob/4b4644870a086db51dbbbfe720d1bb3776be5ba9/src/Billing/Billing/help/Get-AzConsumptionBudget.md#examples

Reply:
Can you be specific what is looking weird?

Reply:
Output is hard to read: 
![Annotation 2019-08-19 133621](https://user-images.githubusercontent.com/11371776/63241533-aa4ba080-c286-11e9-8b16-6647bf7a6da1.png)

I suggest wrap the input and output in one code block, as the example above.